.pre-commit-config.yaml

```diff
@@ -50,7 +50,8 @@ repos:
     hooks:
     -   id: pydocstyle
         # TODO: add packages one by one to enforce pydocstyle eventually
-        files: (^dpbench/config/|^scripts/|^dpbench/console/|^dpbench/infrastructure/benchmark_runner.py|^dpbench/infrastructure/benchmark_validation.py)
+        files: (^dpbench/config/|^scripts/|^dpbench/console/|^dpbench/infrastructure/benchmark_runner.py|^dpbench/infrastructure/benchmark_validation.py|
```

Comment:
please keep files in one line. Second line is actually not tracked. You are missing errors:
```
dpbench/benchmarks/rodinia/gaussian/gaussian_sycl_native_ext/__init__.py:1 at module level:
        D104: Missing docstring in public package
dpbench/benchmarks/rodinia/gaussian/__init__.py:5 at module level:
        D212: Multi-line docstring summary should start at the first line
dpbench/benchmarks/rodinia/gaussian/__init__.py:5 at module level:
        D415: First line should end with a period, question mark, or exclamation point (not 'n')
dpbench/benchmarks/rodinia/gaussian/gaussian_initialize.py:1 at module level:
        D100: Missing docstring in public module
dpbench/benchmarks/rodinia/gaussian/gaussian_initialize.py:8 in public function `initialize`:
        D103: Missing docstring in public function
dpbench/benchmarks/rodinia/gaussian/gaussian_numba_dpex_k.py:1 at module level:
        D100: Missing docstring in public module
dpbench/benchmarks/rodinia/gaussian/gaussian_numba_dpex_k.py:10 in public function `gaussian_kernel_1`:
        D103: Missing docstring in public function
dpbench/benchmarks/rodinia/gaussian/gaussian_numba_dpex_k.py:43 in public function `gaussian_kernel_2`:
        D103: Missing docstring in public function
dpbench/benchmarks/rodinia/gaussian/gaussian_numba_dpex_k.py:72 in public function `gaussian`:
        D103: Missing docstring in public function
dpbench/benchmarks/rodinia/gaussian/gaussian_python.py:1 at module level:
        D100: Missing docstring in public module
dpbench/benchmarks/rodinia/gaussian/gaussian_python.py:6 in public function `gaussian`:
        D103: Missing docstring in public function
```

Reply:
You can keep it multiple lines if you find a way to take care about every line